test/api/v01/isoline_test.rb

```diff
@@ -67,4 +67,13 @@ def test_isoline_limit_should_not_pass
     get '/0.1/isoline', api_key: 'demo_limit', src: '43.2804,5.3806,43.291576,5.355835', dst: '43.2804,5.3806,43.291577,5.355836'
     refute last_response.ok?, last_response.body
   end
+
+  def test_count_isolines
+    [:get, :post].each_with_index do |method, indx|
+      send method, '/0.1/isoline', {api_key: 'demo', loc: '43.2804,5.3806', size: 1}
+      keys = RouterWrapper.config[:redis_count].keys("router:isolines:#{Time.now.utc.to_s[0..9]}_key:demo_ip*")
+      assert_equal 1, keys.size
+      assert_equal({'hits' => (indx + 1).to_s, 'transactions' => (indx + 1).to_s}, RouterWrapper.config[:redis_count].hgetall(keys.first))
```

Comment:
I don't understand how this test can be OK in Count api calls commit...

Reply:
I don't understand why it should not work. There is 2 requests each one has 1 transaction. so first time the count is 1 hit, 1 transaction and second time it's 2 hits, 2 transactions.